wazo_confd/plugins/user_blocklist/schema.py

```diff
@@ -1,12 +1,22 @@
 # Copyright 2025 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
+from __future__ import annotations
+
+import logging
 
 import phonenumbers
 from marshmallow import fields, validates
 from marshmallow.exceptions import ValidationError
 from marshmallow.validate import Length
 
 from wazo_confd.helpers.mallow import BaseSchema, Link, ListLink, number_field
```

Comment:
I didn't catch it in the last PR, but please keep the name consistent with other mallow helpers, i.e. using a class name. Consumers don't need to know if the field is implemented with a function or a class.